x-pack/plugins/actions/server/actions_client/actions_client.ts

```diff
@@ -481,6 +482,13 @@ export class ActionsClient {
     return getAll({ context: this.context, includeSystemActions });
   }
 
+  /**
+   * Get all system connectors
+   */
+  public async getAllSystemActions(): Promise<FindConnectorResult[]> {
```

Comment:
I think the things this method returns are connectors.
Should this be `getAllSystemConnectors` or `getAllSystemActionConnectors` ?

Reply:
You are right! My brain is wired to the old terminology.

Reply:
Done in [`6324edc` (#167783)](https://github.com/elastic/kibana/pull/167783/commits/6324edc9669b74761b4f042eb58baf1414e198d9)